packages/calcite-components/src/components/time-picker/time-picker.e2e.ts

```diff
@@ -1449,4 +1449,47 @@ describe("calcite-time-picker", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(html`<calcite-time-picker></calcite-time-picker>`, {
+        "--calcite-time-picker-background-color": {
+          targetProp: "backgroundColor",
+          shadowSelector: `.${CSS.timePicker}`,
+        },
+        "--calcite-time-picker-corner-radius": {
+          targetProp: "borderRadius",
+          shadowSelector: `.${CSS.timePicker}`,
+        },
+        "--calcite-time-picker-button-background-color-hover": {
+          targetProp: "backgroundColor",
+          state: "hover",
+          shadowSelector: `.${CSS.button}`,
+        },
+        "--calcite-time-picker-button-background-color-press": {
+          targetProp: "backgroundColor",
+          state: { press: { attribute: "class", value: CSS.button } },
+          shadowSelector: `.${CSS.button}`,
+        },
+        "--calcite-time-picker-color": {
+          targetProp: "color",
+          shadowSelector: `.${CSS.timePicker}`,
+        },
+        "--calcite-time-picker-icon-color": {
+          targetProp: "color",
+          shadowSelector: "calcite-icon",
+        },
+        // "--calcite-time-picker-input-border-color-press": {
+        //   targetProp: "boxShadow",
+        //   state: { press: { attribute: "class", value: CSS.hour } },
+        //   shadowSelector: `.${CSS.hour}`,
+        // },
```

Comment:
Does this code need to be reactivated?

Reply:
I have this one commented because I don't think I can test it. I don't think our testing allows to test the color of a box shadow. @macandcheese do you know of a way to test the box shadow color?

Reply:
Fixed!